code/_onclick/item_attack.dm

```diff
@@ -160,6 +160,21 @@
 
 /// The equivalent of [/obj/item/proc/attack] but for alternate attacks, AKA right clicking
 /obj/item/proc/attack_alt(mob/living/victim, mob/living/user, params)
+	if(sharpness == SHARP_POINTY | SHARP_EDGED)
+		force -= force_reduction_stab
+		wound_bonus += wound_bonus_stab
+		armour_penetration += armor_pene_stab
+		attack_verb_continuous = list("stabs")
+		attack_verb_simple = list("stab")
+		sharpness -= SHARP_EDGED
+		attack(victim, user, params)
+		force += force_reduction_stab
+		wound_bonus -= wound_bonus_stab
+		armour_penetration -= armor_pene_stab
+		attack_verb_continuous = initial(attack_verb_continuous)
+		attack_verb_simple = initial(attack_verb_simple)
+		sharpness = initial(sharpness)
+		return
```

Comment:
Wont initial on values here that are lists evaluate to null?

Reply:
I tested it and it works so yeah it doesnt revert to null